README.md

```diff
@@ -1,6 +1,4 @@
-# Record & Tuple
-
-ECMAScript proposal for the Record and Tuple value types.
+# JavaScript Records & Tuples Proposal
 
 **Authors:**
```

Comment:
at this point, you should add yourself as a co-author

Reply:
Eh, let's procrastinate on adding this for now.